templates/dashboard/enter_sudo.html

```diff
@@ -20,6 +20,18 @@ <h1 class="h2">Entering Sudo Mode</h1>
         {{ render_field_errors(password_check_form.password) }}
         <button class="btn btn-lg btn-danger mt-2">Submit</button>
       </form>
+
+    {% if connect_with_proton %}
+      <div class="my-3">
+        <p>Alternatively you can use your Proton credentials to ensure it's you.</p>
+      </div>
+      <a class="btn btn-primary btn-block mt-2 proton-button w-25"
+         href="{{ url_for("auth.proton_login", next=next) }}">
+        <img class="mr-2" src="/static/images/proton.svg" />
+        Login with Proton
```

Comment:
I'd change the wording a bit here to "Authenticate with Proton"

Reply:
Updated.